### PR TITLE
feat: add method add_link to Changelog

### PR DESCRIPTION
**Pull request by gortavoher**
### Description

Add a public add_link method to Changelog to allow the user to add a link to content added to the changelog such as the link to a related issue. 

### Summary of changes

* chore(Cargo.lock): update dependencies getrandom to version 0.2.14 and uuid to version 1.8.0
* feat(Cargo.toml): add uuid dependency with version 1.8.0 and features ["v4"]
* feat(changelog.rs): add method add_link to Changelog struct
* feat(test): add test for adding a link to Changelog
* refactor(parser.rs): change log level from debug to trace in Parser struct
* refactor(token.rs): change log level from debug to trace in tokenize function
* docs(changelog.rs): add example for adding a link to the list of links
* chore(Cargo.lock): update package versions

@napalmpapalam

### CODE REVIEW

File: src/changelog.rs [hunk 1]
```diff
@@ -387,6 +387,30 @@ impl Changelog {
...
+    ///
+    /// changelog.add_link("[anchor]: https://example.com".to_string());
+    ///
+    /// // Assert that the link was added correctly
+    /// assert_eq!(changelog.links().len(), 1);
+    /// assert_eq!(changelog.links().first().unwrap().anchor(), "anchor");
+    /// assert_eq!(changelog.links().first().unwrap().url(),"https://example.com");
+    /// # }    
+    ///     
+    pub fn add_link<T: Into<String>>(&mut self, link: T) -> &mut Self {
```
Comment (napalmpapalam) [inline src/changelog.rs]: mb better to make anchor and link as a separate arguments?

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: src/changelog.rs (hunk 1 above)
Comment (gortavoher) [reply src/changelog.rs]: Updated to collect strings for anchor and url with basic validation to strip brackets and colon if supplied on the anchor and ensure that the strings are not empty. 
@napalmpapalam

**Comment by gortavoher** [comment]
@napalmpapalam  requested change has been implemented.

**Review by napalmpapalam — APPROVED** [review]

**Comment by napalmpapalam** [comment]
Released under the 0.1.4

**Pull request merged**